controller/cmd/service-mirror/events_formatting.go

```diff
@@ -41,14 +41,14 @@ func formatService(svc *corev1.Service) string {
 	return fmt.Sprintf("Service: {name: %s, namespace: %s, annotations: [%s], labels [%s]}", svc.Name, svc.Namespace, formatMetadata(svc.Annotations), formatMetadata(svc.Labels))
 }
 
-func formatEndpoints(endp *corev1.Endpoints) string {
+func formatEndpoints(endpoints *corev1.Endpoints) string {
```

Comment:
This was actually confusing, as one would expect it to not be a plural object based on its name...